app/src/main/java/com/kickstarter/ui/viewholders/UnansweredSurveyHeaderViewHolder.java

```diff
@@ -0,0 +1,48 @@
+package com.kickstarter.ui.viewholders;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.View;
+import android.widget.TextView;
+
+import com.kickstarter.R;
+import com.kickstarter.libs.KSString;
+import com.kickstarter.libs.utils.ObjectUtils;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+public class UnansweredSurveyHeaderViewHolder extends KSViewHolder {
+
+  private final KSString ksString;
+
+  @Bind(R.id.heading) TextView headingTextView;
+
+  public UnansweredSurveyHeaderViewHolder(final @NonNull View view) {
+    super(view);
+    ButterKnife.bind(this, view);
+
+    ksString = environment().ksString();
+  }
+  @Override
+  public void bindData(final @Nullable Object data) throws Exception {
+
+    // Null data means we should hide the view
```

Comment:
instead of `null` I think we should require that `data` be a positive integer. we would never configure this view holder with `0` or `null`, cause we would just omit the view in that case.